package.json

```diff
@@ -19,11 +19,10 @@
   },
   "homepage": "https://github.com/asyncapi/markdown-template#readme",
   "scripts": {
-    "start": "node ./node_modules/@asyncapi/generator/cli.js ./sample.yaml ./ -o ./output --force-write",
+    "start": "node ./node_modules/@asyncapi/generator/cli.js ./test/spec/asyncapi_2.yml ./ -o ./output --force-write",
     "test": "jest --coverage",
     "lint": "eslint --max-warnings 0 --config \".eslintrc\" \".\"",
     "lint:fix": "eslint --max-warnings 0 --config \".eslintrc\" \".\" --fix",
-    "dev": "ag test/spec/asyncapi.yml ./ -o test/output --force-write --watch-template",
```

Comment:
Did not see a reason to have two commands to test with 🤔

Reply:
The `--watch-template` param is helpful when you're developing the Markdown template layout. This way you don't have to execute the command on every single change.